linalg/solvers.hpp

```diff
@@ -169,10 +169,11 @@ class IterativeSolver : public Solver
 
    ///@}
 
-   /// Return the dot product of @a x and @a y
-   double Dot(const Vector &x, const Vector &y) const;
+   /// Return the standard inner (dot) product of @a x and @a y
+   /// Overriding this method in a derived class enables a custom inner product.
```

Comment:
Change the doxygen to `/** */` or add `@brief` to the previous line, because the short documentation has disappeared this way 😮 .